packages/rancher-istio/charts/Chart.yaml

```diff
@@ -14,4 +14,4 @@ annotations:
   catalog.cattle.io/ui-component: istio
   catalog.cattle.io/requires-gvr: monitoring.coreos.com.prometheus/v1
   catalog.cattle.io/provides-gvr: networking.istio.io.virtualservice/v1beta1
-  catalog.cattle.io/auto-install: rancher-kiali-server-crd=1.22.001
+  catalog.cattle.io/auto-install: rancher-kiali-server-crd=1.23.001
```

Comment:
Hmm, maybe specifying which CRD charts you chart is dependent on should be something addressed by https://github.com/rancher/rancher/issues/28730.

Will keep track of it